src/connect.spec.lua

```diff
@@ -248,4 +248,49 @@ return function()
 
 		Roact.unmount(handle)
 	end)
+
+	it("should render parent elements before children", function()
+		local function mapStateToProps(state)
+			return {
+				count = state.count,
+			}
+		end
+
+		local childWasRenderedFirst = false
+
+		local function ChildComponent(props)
+			if props.count > props.parentCount then
+				childWasRenderedFirst = true
+			end
+		end
+
+		local ConnectedChildComponent = connect(mapStateToProps)(ChildComponent)
+
+		local function ParentComponent(props)
+			return Roact.createElement(ConnectedChildComponent, {
+				parentCount = props.count,
+			})
+		end
+
+		local ConnectedParentComponent = connect(mapStateToProps)(ParentComponent)
+
+		local store = Rodux.Store.new(reducer)
+		local tree = Roact.createElement(StoreProvider, {
+			store = store,
+		}, {
+			parent = Roact.createElement(ConnectedParentComponent),
+		})
+
+		local handle = Roact.mount(tree)
+
+		store:dispatch({ type = "increment" })
+		store:flush()
+
+		store:dispatch({ type = "increment" })
+		store:flush()
+
+		Roact.unmount(handle)
+
+		expect(childWasRenderedFirst).to.equal(false)
+	end)
```

Comment:
One more thing came to mind recently when discussing this with @LPGhatguy. I think this test should probably force the config value to `true`, then restore it to what it was before at the end.  Otherwise, this test will fail if the config value is set to false, right?

Reply:
Ah yeah, this test does fail when the config value is set to false. I'll update the test.

Reply:
Might be a good idea to have the inverse test that forces the config to false and tests that.

Reply:
Okay, I just copied and pasted the test (and changed true to false) since this config will be removed soon anyway.